src/phoenix/trace/llama_index/callback.py

```diff
@@ -186,9 +201,7 @@ def __init__(
     ) -> None:
         super().__init__(event_starts_to_ignore=[], event_ends_to_ignore=[])
         self._tracer = Tracer(on_append=callback, exporter=exporter)
-        self._event_id_to_event_data: Dict[CBEventID, CBEventData] = defaultdict(
-            lambda: CBEventData()
-        )
+        self._event_id_to_event_data: EventData = defaultdict(lambda: CBEventData())
```

Comment:
does `EventData` need to be `typing.DefaultDict` instead?

Reply:
I'm not hitting type errors at the moment. Gonna let it slide.